src/target/riscv/riscv.h

```diff
@@ -57,6 +57,14 @@ typedef enum riscv_mem_access_method {
 	RISCV_MEM_ACCESS_MAX_METHODS_NUM
 } riscv_mem_access_method_t;
 
+typedef enum riscv_translation_driver {
+	RISCV_TRANSLATION_DRIVER_HW,
+	RISCV_TRANSLATION_DRIVER_SW,
+	RISCV_TRANSLATION_DRIVER_OFF
+} riscv_translation_driver_t;
```

Comment:
Similarly to the other comments, this can be renamed. For example:

```suggestion
typedef enum riscv_virt2phys_mode {
	RISCV_VIRT2PHYS_MODE_HW,
	RISCV_VIRT2PHYS_MODE_SW,
	RISCV_VIRT2PHYS_MODE_OFF
} riscv_virt2phys_mode_t;
```

Reply:
Addressed